dev/src/index.ts

```diff
@@ -304,7 +299,16 @@ export class Firestore {
     } as AnyDuringMigration);
 
 
-    this.validateAndApplySettings(settings!);
+    const libraryHeader =  {
+      libName: 'gccl',
+      libVersion,
+    };
+
+    if (settings && settings.firebaseVersion) {
+      libraryHeader.libVersion += ' fire/' + settings.firebaseVersion;
```

Comment:
It seems hacky to add another header this way. Is it possible to make an array of headers ?

Reply:
I don't disagree with you, but I don't think this it is possible without changing the generated code. Currently, the generated code only exposes libName and libVersion.

@JustinBeckwith What are your thoughts on this?